compiler/rustc_mir/src/interpret/operand.rs

```diff
@@ -74,14 +74,6 @@ impl<'tcx, Tag> Immediate<Tag> {
     pub fn to_scalar(self) -> InterpResult<'tcx, Scalar<Tag>> {
         self.to_scalar_or_uninit().check_init()
     }
-
-    #[inline]
-    pub fn to_scalar_pair(self) -> InterpResult<'tcx, (Scalar<Tag>, Scalar<Tag>)> {
```

Comment:
Please keep this function, it nicely complements `to_scalar` and might see users again in the future.